cloud

```diff
@@ -907,7 +925,8 @@ function execute_command() {
     # Convert submitters from a string of words to an array
     eval "declare -a submitters=($submitters)"
 
-    declare -r trigger_topic="${prefix}trigger"
+    declare -r trigger_topic="${prefix}load_queue_trigger"
+    declare -r notification_topic="${prefix}pick_notifications_trigger"
```

Comment:
Could you rename the variables and the option arguments to the functions, as well, so they make better sense and it's easier to track them? I.e. change `trigger_topic` to `load_queue_trigger_topic`, and `notification_topic` to `pick_notifications_trigger_topic`?

What you named `notification_topic` is still a "trigger topic" in the sense that it's only used for triggering a function execution, and its messages don't carry any information. And now that we have two "trigger topics", we need to distinguish them somehow too.

That's a lot of repetition, of course, but that's because we're at the start of implementation of our deployment and have little reuse and decoupling, still.